### PR TITLE
add custom controls

### PR DESCRIPTION
**Pull request by dimola7**
Removed the default controls and added custom ones.

### CODE REVIEW

File: src/components/VideoControls/VideoControls.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    }
+    setPauseplayIcon(video.paused ? playIcon : pauseIcon);
+  };
+
+  const muteVideo = () => {
+    if (video.muted) {
+      video.muted = false;
+    } else {
+      video.muted = true;
+    }
```
Comment (ifedapoolarewaju) [inline src/components/VideoControls/VideoControls.js:71]: we can change this block to:

```suggestion
    video.muted = !video.muted;
```

File: src/components/VideoControls/VideoControls.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+  const volumeChange = () => {
+    video.volume = volumeSlider.current.value / 1;
+    setMuteUnmuteIcon(video.volume === 0 ? mutedIcon : speakerIcon);
+  };
+
+  const toggleFullscreen = () => {
+    if (document.fullscreenElement === null) {
+      videoContainer.requestFullscreen();
+      controls.current.style.top = "90%";
```
Comment (ifedapoolarewaju) [inline src/components/VideoControls/VideoControls.js:83]: can this style be done in the css file instead? with the `:fullscreen` css pseudo-class?

**Review by ifedapoolarewaju — COMMENTED** [review]
Thank you for the work! I have dropped some comments and questions. Let me know what you think.

Thank you again!

File: src/App.css [hunk 3]
```diff
@@ -3,7 +3,7 @@
   color: pink;
 }
 
-.App button {
+.App .big-buttons {
```
[the remark this answers is not in the document]
Comment (dimola7) [reply src/App.css:6]: Yeah, I could rename them default-button

File: src/App.js [hunk 4]
```diff
@@ -1,48 +1,60 @@
-import React, { useState } from 'react';
-import logo from './logo.png';
-import VideoSelector from './components/VideoSelector/VideoSelector'
-import SubtitleSelector from './components/SubtitleSelector/SubtitleSelector'
-import VideoPlayer from './components/VideoPlayer/VideoPlayer'
-import Fork from './components/Fork/Fork'
-import './App.css';
+import React, { useState } from "react";
```
[the remark this answers is not in the document]
Comment (dimola7) [reply src/App.js:1]: Its my formatter, I'll change them to single quotes

File: src/App.css [hunk 5]
```diff
@@ -54,11 +54,110 @@
...
+  .controls {
+    width: 90%;
+    top: 49%;
+    left: 49.9%;
+  }
+  .App-name {
+    margin-left: 8px;
+    font-size: 32px;
+  }
+}
```
[the remark this answers is not in the document]
Comment (dimola7) [reply src/App.css:163]: Wow, didn't even take note of that, would transfer them

File: src/components/VideoControls/VideoControls.js [hunk 6]
```diff
@@ -0,0 +1,149 @@
+import React, { useEffect, useState, useRef } from "react";
+import playIcon from "../../play.svg";
+import speakerIcon from "../../volume-up.svg";
+import fullscreen from "../../expand.svg";
+import disableFullscreen from "../../compress.svg";
```
[the remark this answers is not in the document]
Comment (dimola7) [reply src/components/VideoControls/VideoControls.js:5]: Will do that

File: src/components/VideoControls/VideoControls.js (hunk 1 above)
Comment (dimola7) [reply src/components/VideoControls/VideoControls.js:71]: Noted. Will do

File: src/components/VideoControls/VideoControls.js (hunk 2 above)
Comment (dimola7) [reply src/components/VideoControls/VideoControls.js:83]: I'll look into that too

**Pull request merged**